src/app/containers/Header/ScriptLink/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+import ScriptLink from '@bbc/psammead-script-link';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { RequestContext } from '#contexts/RequestContext';
+import { UserContext } from '#contexts/UserContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { getOtherVariant } from '#lib/utilities/variantHandler';
+
+export const generateHref = (service, otherVariant) =>
```

Comment:
This will always go to the homepage. This is not the behaviour that's currently on PAL, where the current page is requested in the other variant.

Reply:
Do we want this in this PR or a seperate issue?

Reply:
Hmm, I think we should generate the correct href in this PR, and then release this version of the script switch.

Reply:
Test assets for checking this behaviour locally: 
Article: http://localhost:7080/zhongwen/articles/c3xd4x9prgyo/simp
CPS asset: http://localhost:7080/zhongwen/chinese-news-49631219/simp
Legacy map urls don't work for variants yet on localhost - will open an issue for this.

Reply:
[Issue](https://github.com/bbc/simorgh/issues/5717) and [PR](https://github.com/bbc/simorgh/pull/5718) for ensuring legacy assets work locally in Simorgh for UKChina and Zhongwen.